algorithms/src/main/kotlin/io/kinference/algorithms/gec/corrector/GECCorrector.kt

```diff
@@ -0,0 +1,191 @@
+package io.kinference.algorithms.gec.corrector
+
+import io.kinference.algorithms.gec.postprocessing.GecPostprocessor
+import io.kinference.algorithms.gec.preprocessing.*
+import io.kinference.algorithms.gec.utils.*
+import kotlin.math.min
+
+/**
+ * data class for Correction result from each iteration
+ */
+data class CorrectionResult(val sentence: String, val corrections: SentenceCorrections)
```

Comment:
Maybe move it inside `GECCorrector`? If the result is by `GECCorrector` then it should be nested class.